localgov_news.install

```diff
@@ -14,4 +14,19 @@ function localgov_news_install() {
   if (\Drupal::moduleHandler()->moduleExists('user')) {
     user_role_grant_permissions(RoleInterface::ANONYMOUS_ID, ['use search_api_autocomplete for localgov_news_search']);
   }
+
+  // Install default config for simple_sitemap, as this does not appear to work
+  // in the config/optional folder.
+  // Discussed on https://www.drupal.org/project/simple_sitemap/issues/3156080
+  if (\Drupal::moduleHandler()->moduleExists('simple_sitemap')) {
```

Comment:
Should we also check here to see if a site has already set up a site map for the news bundles? BHCC has already (although it uses the same settings as below)

Reply:
Probably a good idea. This will change the priority if you've enabled this config already with a different priority to the default.

Reply:
Having taken another look at this, the sitemap config should only be added when the News module is installed, so we don't need to check if the config already exists.

One issue I found was there was no `$is_syncing` check so this would have overwritten previous config. This is now fixed.

Reply:
On Merge monday, @ekes agrees that this will only happen on install.
We can make a followup to add an hook_installed in case metatag is installed after localgov_news.